dagmc_stats/DagmcStats.py

```diff
@@ -48,6 +41,9 @@ def __init__(self, filename, populate=False):
         self.__set_dagmc_tags()
         self.entityset_ranges = {}
         self.__set_entityset_ranges()
+        
+        if populate is True:
+            self.__populate_triangle_data(meshset)
```

Comment:
move the creation of the data frame after this line so it can be populated all at once and save time